contracts/src/Types.sol

```diff
@@ -45,7 +45,8 @@ struct InboundMessage {
 
 enum OperatingMode {
```

Comment:
Assuming we do go ahead with this change (see above comment), you must also make sure update https://github.com/Snowfork/snowbridge/blob/3dad79b95b0ee26d8e8f62973e071a767a386d5b/parachain/primitives/core/src/outbound.rs#L53

Reply:
It's coverd in https://github.com/Snowfork/snowbridge/pull/917/commits/abf2f97dac370b59e4a34ee9574185067041ff12